globus_automate_client/flows_client.py

```diff
@@ -194,7 +204,7 @@ def deploy_flow(
         """
         if validate_definition:
             validate_flow_definition(flow_definition)
-        self.authorizer = get_authorizer_for_scope(MANAGE_FLOWS_SCOPE, self.client_id)
+        self.authorizer = self.flow_management_authorizer
```

Comment:
This should just get set on the super class during `__init__` and thus we don't have to futz with setting `self.authorizer` any place else.

Reply:
Oops, replying to myself, maybe not. I'll have to think ahead. When we do `run` do we need a different scope on the Flows service or are we then using the scope on the Flow Id. Probably the latter, so maybe we do need this.

Hmm, need to think a bit here on how to make this as simple as possible. I guess the callback handles it ok, but in the perhaps common case where a user creates one `FlowClient` and then does a lookup and then a run sort of thing would be nice not to have to juggle authorizers or call the callback multiple times which may, in turn, cause the user to go through consent more than once.

I don't have the solution on this one right away...